sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -26,8 +26,16 @@ def data():
 
 @pytest.fixture(scope="module")
 def data_binary(data):
-    X, y = data
-    return X[y < 2], y[y < 2]
+    X, y = make_classification(
+        n_samples=200,
+        n_features=20,
+        n_informative=5,
```

Comment:
Not sure if we need that many features (and so many uninformative ones), but I will leave to another maintainer to determine.